pkg/ds/daemon_set.go

```diff
@@ -559,11 +554,10 @@ func (ds *daemonSet) PublishToReplication() error {
 	// InitializeReplicationWithCheck, we will get an error
 	ds.cancelReplication()
 
-	podLocations, err := ds.CurrentPods()
+	nodes, err := ds.EligibleNodes()
```

Comment:
i'm hoping this doesn't have unintended consequences. Prior to this commit we would label the new nodes ahead of time and then target the labeled nodes (aka `CurrentPods()`). Now they're not labeled yet, so we target the eligible ones (that match the node selector).

99% sure it's the same thing but check my work on that